packages/math-input/src/components/keypad/mobile-keypad-internals.tsx

```diff
@@ -180,7 +181,11 @@ class MobileKeypadInternals
         const convertDotToTimes = keypadConfig?.times;
 
         return (
-            <View style={containerStyle} forwardRef={this._containerRef}>
+            <View
+                data-testid="perseus-mobile-keypad"
```

Comment:
Can we remove this? React Testing Library really recommends avoiding usage of test IDs if at all possible. Given there are no tests in this PR it seems like test ID is not needed here.